flaskr/app.py

```diff
@@ -36,7 +39,7 @@ def create_app(env_name):
     return app
 
 if __name__ == '__main__':
-    # env_name = os.getenv('FLASK_ENV')
-    env_name = 'development'
-    app = create_app(env_name)
+    environment_name = os.getenv('FLASK_ENV')
```

Comment:
no default environment ?

Reply:
I have changed so development is the default one